### ISSUE TITLE
ListViewItem.ListViewItemAccessibleObject.ScrollIntoView uses ListBox messages

### ISSUE THREAD

**Issue by Grammarian** [dotnet/winforms#3957]
* .NET Core Version: 
Microsoft.WindowsDesktop.App 5.0.0-rc.1.20452.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

* Have you experienced this same bug with .NET Framework?: 
No

**Problem description:**

In the accessibility object for ListViewItem, the method `ScrollIntoView()` uses messages and techniques for a ListBox, not a ListView.

* Messages sent to the ListView are from the `User32.LB` enum.

* The code tries to calculate the height of each item in the control, which is needed for ListBox, but does not work for ListView.

`.\System.Windows.Forms\src\System\Windows\Forms\ListViewItem.ListViewItemAccessibleObject.cs`

**Comment by RussKie** [comment]
@Grammarian thank you!

**Comment by RussKie** [comment]
@vladimir-krestov this appears to be the only place of incorrect use of LB flags:
![image](https://user-images.githubusercontent.com/4403806/93732312-18922d80-fc14-11ea-8e10-063f5885456b.png)

**Comment by merriemcgaw** [comment]
This is fixed for 5.0 RC2 and needs to be ported.

**Comment by SergeySmirnov-Akvelon** [comment]
Fix is [ported](https://github.com/dotnet/winforms/commit/6391b524dd64e7530eba986c6607bf129b3ec8ae) to master  branch

**Comment by Jenna-Zheng** [comment]
Verified this issue with .NET Core SDK 5.0.100-rtm.20513.7, it is fixed that this method starts to work correctly and user or program have ability to see required element after calling of "ScrollIntoView" method. please see the following gif.
![TestResult_focus](https://user-images.githubusercontent.com/64949616/95951410-f2ce0200-0e28-11eb-8fa2-b9e20316bd45.gif)
